### PR TITLE
Add pytest-virtualenv recipe

### PR DESCRIPTION
**Pull request by jcfr**
See https://github.com/manahl/pytest-plugins/tree/master/pytest-virtualenv#readme

### CODE REVIEW

**Comment by jcfr** [comment]
@marcelotrevisani  Et voila, here is the recipe that depends on the previous two packages you helped me integrate.

File: recipes/pytest-virtualenv/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+requirements:
+  host:
+    - pip
+    - pip
+    - pytest
+    - pytest-fixture-config
+    - pytest-shutil
+    - python
+    - setuptools
```
Comment (marcelotrevisani) [inline recipes/pytest-virtualenv/meta.yaml]: please add
```
- setuptools-git
```
as a dependency as well

File: recipes/pytest-virtualenv/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - pip
```
Comment (marcelotrevisani) [inline recipes/pytest-virtualenv/meta.yaml]: you put ``pip`` twice as requirement, please remove one

File: recipes/pytest-virtualenv/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    - pytest-fixture-config
+    - pytest-shutil
+    - python
+    - setuptools
+  run:
+    - pytest
+    - pytest-fixture-config
+    - pytest-shutil
+    - python
+    - setuptools
```
Comment (marcelotrevisani) [inline recipes/pytest-virtualenv/meta.yaml]: please remove setuptools from here

File: recipes/pytest-virtualenv/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+test:
+  imports:
+    - pytest_virtualenv
+  requires:
```
Comment (marcelotrevisani) [inline recipes/pytest-virtualenv/meta.yaml]: please add ``mock`` as test requirement

**Comment by jcfr** [comment]
Thanks for the review :+1:

**Comment by marcelotrevisani** [comment]
Hi @jcfr thanks to put that in ``conda-forge`` !
I'm going to merge that!

I just saw you work for kitware right!? I did a PR for them in the end of the last year regarding the convertion ``numpy`` to ``vtk`` arrays, :)

**Comment by jcfr** [comment]
Thanks for merging.

>  just saw that you work for kitware right!? 

Indeed, I am in the North Carolina office, let me know if you end up passing by.

> I did a PR for them in the end of the last year regarding the convertion numpy to vtk arrays, :)

Thanks for contributing :smile:

**Pull request merged**